Mathlib/NumberTheory/DiophantineApproximation.lean

```diff
@@ -369,22 +368,20 @@ theorem convergent_of_int {ξ : ℤ} (n : ℕ) : convergent ξ n = ξ := by
 #align real.convergent_of_int Real.convergent_of_int
 
 /-!
-Our `convergent`s agree with `GeneralizedContinuedFraction.convergents`.
+Our `convergent`s agree with `GenContFract.convs`.
 -/
 
 
-open GeneralizedContinuedFraction
+open GenContFract
 
-/-- The `n`th convergent of the `GeneralizedContinuedFraction.of ξ`
-agrees with `ξ.convergent n`. -/
-theorem continued_fraction_convergent_eq_convergent (ξ : ℝ) (n : ℕ) :
-    (GeneralizedContinuedFraction.of ξ).convergents n = ξ.convergent n := by
+/-- The `n`th convergent of the `GenContFract.of ξ` agrees with `ξ.convergent n`. -/
+theorem genContFract_convs_eq_convergent (ξ : ℝ) (n : ℕ) :
```

Comment:
Sorry, my earlier suggestion here may have been unclear since I edited it after I submitted it. If we keep the `genContFract`, then this should be 
```suggestion
theorem genContFractConvs_eq_convergent (ξ : ℝ) (n : ℕ) :
```
per the naming convention doc I mentioned in a later comment.

Alternatively, as Michael Stoll suggested, this shorter version could also work:
```suggestion
theorem convs_eq_convergent (ξ : ℝ) (n : ℕ) :
```

(And similarly for `exists_genContFract_convs_eq_rat` below.)

Anyways, once you pick one of these, feel free to merge!
bors d+